pyphare/pyphare/pharesee/run/run.py

```
@@ -103,30 +105,69 @@ def GetE(self, time, merged=False, interp="nearest", all_primal=True, **kwargs):
-    def GetFlux(self, time, pop_name, merged=False, interp="nearest", **kwargs):
+    def GetFlux(self, time, pop_name, merged=False, interp="nearest", all_primal=True, **kwargs):
+        if merged:
+            all_primal = False
         hier = self._get_hierarchy(time, f"ions_pop_{pop_name}_flux.h5", **kwargs)
+
+        hier_ = rename(hier, ['Fx', 'Fy', 'Fz'])
+        # print(hier.quantities())
+        if not all_primal:
+            return self._get(hier_, time, merged, interp)
+
+        h = compute_hier_from(_compute_to_primal, hier_, x="Fx", y="Fy", z="Fz")
```

Ruff (0.14.14)

[error] Local variable `h` is assigned to but never used

Remove assignment to unused variable `h`

(F841)